prowler/providers/kubernetes/kubernetes_provider.py

```diff
@@ -299,8 +307,7 @@ def get_all_namespaces(self) -> list[str]:
             logger.critical(
                 f"{error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
-            # TODO: add custom exception once we have the Kubernetes exceptions
-            raise error
+            raise KubernetesGetAllNamespacesError(original_exception=error)
```

Comment:
We should catch the possible Kubernetes APIs exceptions here and see how to handle them.

Reply:
Done